Generate the patch: `git diff --output=../vendor/nlopt.patch`
diff --git a/ports/nlopt/portfile.cmake b/ports/nlopt/portfile.cmake
index 2270ce0182..b4d198244b 100644
--- a/ports/nlopt/portfile.cmake
+++ b/ports/nlopt/portfile.cmake
@@ -11,7 +11,7 @@ vcpkg_cmake_configure(
     OPTIONS
         -DNLOPT_FORTRAN=OFF
         -DNLOPT_GUILE=OFF
-        -DNLOPT_LUKSAN=OFF
+        -DNLOPT_LUKSAN=ON
         -DNLOPT_MATLAB=OFF
         -DNLOPT_OCTAVE=OFF
         -DNLOPT_PYTHON=OFF
